ui/utils.py

```diff
@@ -0,0 +1,12 @@
+"""ui utilities"""
+from enum import (
+    Enum,
+    unique,
+)
+
+
+# Python 3.6: this should be upgraded to enum.Flag
+@unique
+class FeatureFlag(Enum):
+    """FeatureFlag enum"""
+    EXAMS = 1
```

Comment:
Values should be powers of 2 so they can be used in the bitmask, I'll add that explanation to the docstring.